packages/@storefront/core/test/unit/core/url-beautifier/details/parser.ts

```diff
@@ -97,6 +97,36 @@ suite('DetailsUrlParser', ({ expect }) => {
     expect(parsed.variants).to.eql(expectedDetail.variants);
   });
 
+  it('should extract query string parameters of a URL specifying "area" and add it to the return object', () => {
```

Comment:
The test cases above don't include the trailing "...and add it to the return object", so I think we can omit that text here.

Also, I think there's a singular/plural mismatch ("...query string parameters..." vs. "URL specifying 'area'...").  This should probably read something to the effect of:

'should extract and apply the 'area' query string parameter'